build.gradle

```diff
@@ -1,13 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
```

Comment:
I have not changed this file as part of "static scan issues fixes".
But I'm seeing the diff?

Can you confirm on why this came?
Because these changes were part of codebase even during 0.1.1 also

Reply:
I see that this change was not part of 0.1.1 as part of the release.